samples/extensions/shader_debugprintf/CMakeLists.txt

```diff
@@ -31,4 +31,4 @@ add_sample(
     SHADER_FILES_HLSL
         "shader_debugprintf/hlsl/scene.vert.hlsl"
         "shader_debugprintf/hlsl/scene.frag.hlsl"
-    DXC_ADDITIONAL_ARGUMENTS "-fspv-extension=SPV_KHR_non_semantic_info")
```

Comment:
What's the change here?

Reply:
This spurious change is likely caused by a CRLF difference, where I initially made a change in one commit and then reverted.  I suspect editing on macOS resulted in an end-of-line terminator change.  You can either ignore this one or remove from the change list.

Reply:
So, yes, you should remove this from the PR.

Reply:
Will do.